Sources/Playlist/PKPlaylistController.swift

```diff
@@ -0,0 +1,484 @@
+//
+//  PKPlaylistController.swift
+//  KalturaPlayer
+//
+//  Created by Sergii Chausov on 01.09.2021.
+//
+
+import Foundation
+import PlayKit
+
+@objc public class PKPlaylistController: NSObject, PlaylistController {
```

Comment:
Please rename to KPPlaylistController (cause this class is in KalturaPlayer and not PlayKit).
Please extract all code for OTT and OVP and set them in KPOTTPlaylistController and KPOVPPlaylistController accordingly, otherwise none will compile separately.

I will go over this class after it is fixed.